README.md

```diff
@@ -12,15 +12,15 @@
 <p align="center">
   <img src="https://raw.githubusercontent.com/EthVM/ethvm/develop/.github/assets/capture-1.png" width="100" />
   <img src="https://raw.githubusercontent.com/EthVM/ethvm/develop/.github/assets/capture-2.png" width="100" />
-  <img src="https://raw.githubusercontent.com/EthVM/ethvm/develop/.github/assets/capture-3.png" width="100" /> 
+  <img src="https://raw.githubusercontent.com/EthVM/ethvm/develop/.github/assets/capture-3.png" width="100" />
   <img src="https://raw.githubusercontent.com/EthVM/ethvm/develop/.github/assets/capture-4.png" width="100" />
   <img src="https://raw.githubusercontent.com/EthVM/ethvm/develop/.github/assets/capture-5.png" width="100" />
 </p>
 
 ## About
 
 ### What is EthVM?
-EthVM is an open-source [Ethereum](https://www.ethereum.org/) Blockchain Data Processing and Analytics Engine with a client-side Block Explorer, although other networks and forks will be supported over time. EthVM is build by [MyEtherWallet](https://github.com/MyEtherWallet) team and is secured under the [SSPL license](https://www.mongodb.com/licensing/server-side-public-license) (a small variation of the GNU Affero License v3) and is written in a mixture of different languages.
+EthVM is an open-source [Ethereum](https://www.ethereum.org/) Blockchain Data Processing and Analytics Engine with a client-side Block Explorer, although other networks and forks will be supported over time. EthVM is built by the [MyEtherWallet](https://github.com/MyEtherWallet) team and is secured under the [SSPL license](https://www.mongodb.com/licensing/server-side-public-license) (a small variation of the GNU Affero License v3) and is written in a mixture of different languages.
```

Comment:
major change is here:

<b>previously<b>
> EthVM is build by [MyEtherWallet](https://github.com/MyEtherWallet) team ...

<b>now</b>
>   EthVM is built by the [MyEtherWallet](https://github.com/MyEtherWallet) team ...

Reply:
Great Catch! is it ready to be merged or you have other fixes?

Reply:
yup ready for merge!